app/Model/Server.php

```diff
@@ -530,6 +530,14 @@ public function __construct($id = false, $table = null, $ds = null)
                                 'test' => 'testForEmpty',
                                 'type' => 'string',
                         ),
+                        'cweurl' => array(
+                                'level' => 1,
+                                'description' => __('Turn Weakness type attributes into links linking to the provided CWE lookup'),
+                                'value' => '',
```

Comment:
as value, put the URL that the user should expect. This is not acted on, it's display only to describe the default behaviour.

'http://cve.circl.lu/cwe/'